### ISSUE TITLE
Turn off nette/di-specific inspections in XML configuration

### ISSUE THREAD

**Issue by mabar** [mesour-intellij-plugins/neon-nette-issues#40]
Neon is often used as a simple data list, usually for translation messages. In that case it does not make sense to autocomplete PHP or check %parameter% (symfony/translation uses %parameter% for message variables). It would be nice to have xml configuration defining which folders should have nette/di inspections turned off.

I would also like to generate configuration automatically from translator configuration in the app. That would need plugin to detect neon xml config changes and offer reload in runtime, without restarting IDE.

Best case scenario would be to not just turn off inspections, but also be able to specify language used in neon values. That could be used for highlighting and autocomplete of ICU message format (standard for translation messages)

**Comment by lulco** [comment]
Yes! and once we have defined which files are used for translation, it would be super cool, if I could link from neon message to the php / latte code where it is used and vice versa O:-)

**Comment by mabar** [comment]
That's out of scope of neon plugin, but I have that idea for another plugin in mind too :)
